packages/framework/src/hooks/useLaungaugeScope.tsx

```diff
@@ -0,0 +1,8 @@
+import { useTranslation } from "react-i18next";
+
+export interface CustomLanguageScope {
+  [key: string]: unknown;
+}
+export const useLanguageScope = () => {
```

Comment:
Yes, it is used to change the language

Reply:
Where is it being used? Why is this not just part of the application context?

Reply:
This is used in useEnsembleAction to change the current language. We don't need to pass this throughout the entire application since the application context only returns the appAtom value. We just need the changeLanguage function from useTranslation. So, it's better to call this function only when necessary. Additionally, if we need to make any translation-related changes in the future, we can use this hook.

Reply:
I don't think that's the case I don't see usages of this anywhere after checking out this code out locally. Either make everything go through this hook (and fix the spelling of the file) or get rid of it.